src/igvfd/schemas/variant.json

```diff
@@ -0,0 +1,138 @@
+{
+    "title": "Variant",
+    "description": "Schema for submitting a variant.",
+    "$id": "/profiles/variant.json",
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "type": "object",
+    "required": [
+        "ref",
+        "alt",
+        "locations"
+    ],
+    "identifyingProperties": [
+        "uuid",
```

Comment:
Was there some discussion that uuid is redundant in the identifyingProperties field?

Reply:
I don't recall, but every object has it in the identifyingProperties